bindings/java/src/org/sleuthkit/datamodel/WindowsAccountUtils.java

```diff
@@ -110,20 +114,20 @@
 	
 	// Some windows SID indicate special account.
 	// These should be handled differently from regular user accounts.
-	private static final Set<String> SPECIAL_SIDS = ImmutableSet.of(
-			"S-1-5-18",	// LOCAL_SYSTEM_ACCOUNT
-			"S-1-5-19", // LOCAL_SERVICE_ACCOUNT
-			"S-1-5-20" // NETWORK_SERVICE_ACCOUNT
-	);
-	private static final Set<String> SPECIAL_SID_PREFIXES = ImmutableSet.of(
-			"S-1-5-80",	// Virtual Service accounts
-			"S-1-5-82", // AppPoolIdentity Virtual accounts. 
-			"S-1-5-83", // Virtual Machine  Virtual Accounts.
-			"S-1-5-90", // Windows Manager Virtual Accounts. 
-			"S-1-5-96" // Font Drive Host Virtual Accounts.
-	);
-	
-	
+	private static final Map<String, String> SPECIAL_SIDS_MAP =  ImmutableMap.<String, String>builder() 
+			.put("S-1-5-18", "Local System Account")
+			.put("S-1-5-19", "Local Service Account")
+			.put("S-1-5-20", "Network Service Account")
+			.build();
+		
+	private static final Map<String, String> SPECIAL_SID_PREFIXES_MAP = ImmutableMap.<String, String>builder() 
+			.put("S-1-5-80", "All Services")
+			.put("S-1-5-82", "IIS AppPool")
```

Comment:
Id add Virtual Account to the end for consistency with the other virtual accounts